### PR TITLE
Fix branch reference for Rolling

### PR DESCRIPTION
**Pull request by fmauch**
The README referenced a branch called `rolling` as branch for rolling, which doesn't exist. This commit references the `master` branch instead.

### CODE REVIEW

**Comment by christophfroehlich** [comment]
This is linked with https://github.com/ros-controls/realtime_tools/issues/93 -> `humble` branch doesn't exist too.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ See [control.ros.org](http://control.ros.org) and [realtime_tools](http://wiki.r
...
 **Humble** | [`humble`](https://github.com/ros-controls/realtime_tools/tree/humble) | [![Humble Binary Build](https://github.com/ros-controls/realtime_tools/actions/workflows/humble-binary-build-main.yml/badge.svg?branch=master)](https://github.com/ros-controls/realtime_tools/actions/workflows/humble-binary-build-main.yml?branch=master) <br /> [![Humble Semi-Binary Build](https://github.com/ros-controls/realtime_tools/actions/workflows/humble-semi-binary-build-main.yml/badge.svg?branch=master)](https://github.com/ros-controls/realtime_tools/actions/workflows/humble-semi-binary-build-main.yml?branch=master) |  | [realtime_tools](https://index.ros.org/p/realtime_tools/#humble)
```
Comment (christophfroehlich) [inline README.md:10]:
```suggestion
**Humble** | [`master`](https://github.com/ros-controls/realtime_tools/tree/master) | [![Humble Binary Build](https://github.com/ros-controls/realtime_tools/actions/workflows/humble-binary-build-main.yml/badge.svg?branch=master)](https://github.com/ros-controls/realtime_tools/actions/workflows/humble-binary-build-main.yml?branch=master) <br /> [![Humble Semi-Binary Build](https://github.com/ros-controls/realtime_tools/actions/workflows/humble-semi-binary-build-main.yml/badge.svg?branch=master)](https://github.com/ros-controls/realtime_tools/actions/workflows/humble-semi-binary-build-main.yml?branch=master) |  | [realtime_tools](https://index.ros.org/p/realtime_tools/#humble)
```

**Review by christophfroehlich — COMMENTED** [review]
This is linked with https://github.com/ros-controls/realtime_tools/issues/93 -> humble branch doesn't exist too.

**Review by bmagyar — APPROVED** [review]
Police has been dispatched to assess these wrongdoings

**Pull request merged**